### ISSUE TITLE
Speed up travis

### ISSUE THREAD

**Issue by josojo** [gnosis/dex-services#697]
I think it would be valuable to get a quicker travis build, after introducing another docker build with the open-solver

There are two options:
1) building docker images in parallel
2) Using Felix comment:
```
I noticed that we are running this step twice although we probably wouldn't have to (e.g. if we built the binary once and then just copied it into an image that combines the solver base and the binary, but maybe that's out of scope for now).
```

---
I would not have time for it this week, but could tackle it next week.

**Comment by fleupold** [comment]
Covered by #845
